README.rst

```diff
@@ -95,26 +85,23 @@ Testing
 -------
 
 If you've followed the procedure above, you already have all the development
-requirements installed, and you can simply run the test suite:
+requirements installed, and you can run the full test suite (including tests for code style and documentation) with tox_:
 
 .. code-block:: text
 
-    py.test
-
-Please ensure that all patches are accompanied by matching changes in the test
-suite. The project tries to maintain 100% test coverage.
-
-You can also use `tox` to run the full suite of tests, including a quick test
-to check documentation and code linting.
+    tox
 
-The following tox environments are relevant for local testing:
+For speedier testing, we recommend you run `py.test`_ directly on individual test files or folders:
 
 .. code-block:: text
 
-    tox -e py35  # runs all tests with Python 3.5
-    tox -e docs  # runs a does-it-compile check on the documentation
-    tox -e lint  # runs the linter for coding style checks
+    cd test/mitmproxy/addons
+    py.test --cov mitmproxy.addons.anticache --looponfail test_anticache.py
```

Comment:
The "official" usage is actually spelled `pytest` when invoked via CLI.
At least thats what the py.test documentation is using.